app/src/main/java/com/cornellappdev/android/eatery/ui/screens/FavoritesScreen.kt

```diff
@@ -119,24 +176,68 @@ fun FavoritesScreen(
                         }
                     }
                 } else {
+                    Column(
+                        modifier = Modifier.fillMaxWidth(),
+                        verticalArrangement = Arrangement.spacedBy(12.dp),
+                        horizontalAlignment = Alignment.CenterHorizontally
+                    ) {
+                        Row(
+                            horizontalArrangement = (Arrangement.spacedBy(8.dp))
+                        ) {
+                            //toggle.value = true means that the active toggle should be the Eatery button
+                            if (toggle.value) {
+                                ActiveToggle(onClick = { }, label = "Eateries")
+                                InactiveToggle(
+                                    onClick = { toggleViewModel.toggle() },
+                                    label = "Items"
+                                )
+                            } else {
+                                InactiveToggle(
+                                    onClick = { toggleViewModel.toggle() },
+                                    label = "Eateries"
+                                )
+                                ActiveToggle(onClick = { }, label = "Items")
+                            }
```

Comment:
Okay yeah so once you get to the refactor of this, you should then be able to make this NOT AT ALL need an if/else statement